src/api/status/src/services.js

```diff
@@ -9,6 +9,11 @@ const prodUrl = (pathname) => `${prodHost}/${apiVersion}${pathname}`;
 
 // Our current list of microservices
 const services = [
+  // {
```

Comment:
What's happening here?  If this isn't ready to land, don't include it.

Reply:
I'll remove it